<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryPage</class>
 <widget class="QWidget" name="GeometryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Geometry</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSubtitle">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Areas, lateral areas, and volumes of geometric shapes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabArea">
      <attribute name="title">
       <string>Area</string>
      </attribute>
      <layout class="QVBoxLayout" name="areaLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="labelAreaShape">
           <property name="text">
            <string>Shape</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboAreaShape">
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Circle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rectangle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cube</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cuboid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cylinder</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sphere</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAreaInputs">
         <property name="title">
          <string>Inputs</string>
         </property>
         <layout class="QFormLayout" name="areaFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelAreaInput1">
            <property name="text">
             <string>Input 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditAreaInput1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelAreaInput2">
            <property name="text">
             <string>Input 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditAreaInput2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelAreaInput3">
            <property name="text">
             <string>Input 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditAreaInput3"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAreaCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAreaResult">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Area Result</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="labelAreaResult">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>—</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLateral">
      <attribute name="title">
       <string>Lateral Area</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Shape</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboLateralShape">
           <item>
            <property name="text">
             <string>Cone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pyramid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cube</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cuboid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cylinder</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Inputs</string>
         </property>
         <layout class="QFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_la_area1">
            <property name="text">
             <string>Input 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="linelaArea1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_la_area2">
            <property name="text">
             <string>Input 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="linelaArea2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_la_area3">
            <property name="text">
             <string>Input 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="linelaArea3"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonLateralCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Lateral Area Result</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="label_la_result">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>—</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVolume">
      <attribute name="title">
       <string>Volume</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Shape</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboVolumeShape">
           <item>
            <property name="text">
             <string>Cylinder</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sphere</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cuboid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cube</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pyramid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cone</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Inputs</string>
         </property>
         <layout class="QFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_vl1">
            <property name="text">
             <string>Input 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="line_vl_edit1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_vl2">
            <property name="text">
             <string>Input 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="line_vl_edit2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_vl3">
            <property name="text">
             <string>Input 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="line_vl_edit3"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonVolumeCalculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Volume Result</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="label_vl_result">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>—</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
